browserify.js

```diff
@@ -1,8 +1,14 @@
-var THREE = require('three');
-var work = require('webworkify');
-var PhysijsWorker = work(require('./physijs_worker.js'));
+class work {
+	constructor(){
+		const work = require('webworkify');
+		return work;
+	}
+}
 
-// inject Three.js and Physijs's Worker
-var Physijs = require('./physi.js')(THREE, PhysijsWorker);
+const THREE = require('three'),
+	  PhysijsWorker = new work(require('./physijs_worker.js')),
```

Comment:
This doesn't use the new class and hasn't changed what-so-ever -- you just changed the formatting (which should be kept consistent as well)

Reply:
Oh my b, this does actually use the new class because it has the same name as the import. But this shouldn't work anyway because the `work` constructor doesn't take an argument. It would be `work()(require('./physijs_worker.js')`, but that would run into the same "not a constructor issue" -- needs to be required in the constructor per my [comment above](https://github.com/agilgur5/physijs-webpack/pull/3/files#r293201093) to get around this.

Even if that were fixed, it would still run into the next issue I found in https://github.com/agilgur5/physijs-webpack/issues/4#issuecomment-501518040 . #5 fixes that with the `browserify-worker-stub`.

#5 is good to go anyway, so no need for any redundant work here, but just wanted to note this here as I was looking it over again since you said the `Scene` worked with this PR too (the constructor does exist here, it just doesn't return a Worker with anything in it).